virttest/qemu_storage.py

```diff
@@ -1096,6 +1096,129 @@ def snapshot_apply(self):
 
         process.run(cmd)
 
+    def bitmap_add(self, bitmap_name):
+        """
+        Add a bitmap to a image.
+
+        :param bitmap_name: The name of the bitmap
+        """
+
+        bitmap_params = self.params.object_params(bitmap_name)
+        granularity = bitmap_params.get('bitmap_granularity')
+
+        cmd = self.image_cmd
+        if self.image_format == 'qcow2':
+            cmd += " bitmap --add -f %s %s %s" % (
+                self.image_format, self.image_filename, bitmap_name)
+            if granularity:
+                cmd += ' -g %s' % granularity
+        else:
+            raise exceptions.TestError(
+                "Image format %s is not qcow2, check please." %
+                self.image_format)
```

Comment:
```suggestion
                "Image format %s is not supported, check please." %
                self.image_format)
```
I am afraid that will support other image format type in the further, so suggest making it cover such situation.